serverless/main.mk

```diff
@@ -1,26 +1,39 @@
 # Macroses
 ########################################################################################################################
+PROJ_PATH ?= $(PROJ_DIR)/$(SLS_SERVICE)-$(SLS_FUNCTION)
+
+SLS_DOCKER_IMAGE ?= amaysim/serverless
 SLS_VERSION ?= 1.73.1
-SLS_FILE ?= serverless.yml
```

Comment:
Why are we removing SLS_FILE macro (sometimes there are multiple serverless.yml, like `serverless.local.yml`)?

Reply:
fixed